code/game/objects/structures/transit_tubes/station.dm

```diff
@@ -128,30 +132,49 @@
 /obj/structure/transit_tube/station/proc/launch_pod()
 	if(launch_cooldown >= world.time)
 		return
+
+	var/SleepTime = CLOSE_DURATION + 2
 	for(var/obj/structure/transit_tube_pod/pod in loc)
 		if(!pod.moving)
-			pod_moving = 1
+			pod_moving = TRUE
 			close_animation()
-			sleep(CLOSE_DURATION + 2)
+			if (pod.cargo)
+				SleepTime -= 2 //Cargo pods leave the station faster.
+				var/atom/input = get_step(src, turn(boarding_dir, 180))
+				if(pod.contents.len)
+					if(!isopenturf(input)) //Check to make sure the loaded crate doesn't load into a wall
+						input = loc
+					pod.empty_pod(input)
+					playsound(src, 'sound/mecha/mechturn.ogg', 25 ,1)
+				else
+					for (var/obj/structure/S in input)
+						if (istype(S, /obj/structure/closet) && !S.anchored)
+							S.forceMove(pod)
+							playsound(src, 'sound/mecha/mechturn.ogg', 25 ,1)
```

Comment:
```suggestion
					for (var/obj/structure/closet/S in input)
						if (S.anchored)
							continue
						S.forceMove(pod)
						playsound(src, 'sound/mecha/mechturn.ogg', 25 ,1)
```

Reply:
I tried this. It refused to load crates. Doesn't work

Reply:
Theres no reason for that to not work...

Reply:
Ok smartass but it didn't. Searching for(var/obj/struture/closet/S) ONLY gives closets. Not any children.